pgq/snb-load.sql

```diff
@@ -0,0 +1,142 @@
+CREATE OR REPLACE TABLE Comment_hasTag_Tag        AS FROM read_csv('PATHVAR/Comment_hasTag_Tag.csv');
+CREATE OR REPLACE TABLE Comment_isLocatedIn_Country        AS FROM read_csv('PATHVAR/Comment_isLocatedIn_Country.csv');
+CREATE OR REPLACE TABLE Comment_replyOf_Comment        AS FROM read_csv('PATHVAR/Comment_replyOf_Comment.csv');
+CREATE OR REPLACE TABLE Comment_replyOf_Post        AS FROM read_csv('PATHVAR/Comment_replyOf_Post.csv');
+CREATE OR REPLACE TABLE Forum_containerOf_Post        AS FROM read_csv('PATHVAR/Forum_containerOf_Post.csv');
+CREATE OR REPLACE TABLE Forum_hasMember_Person        AS FROM read_csv('PATHVAR/Forum_hasMember_Person.csv');
+CREATE OR REPLACE TABLE Forum_hasModerator_Person        AS FROM read_csv('PATHVAR/Forum_hasModerator_Person.csv');
+CREATE OR REPLACE TABLE Forum_hasTag_Tag        AS FROM read_csv('PATHVAR/Forum_hasModerator_Person.csv');
+CREATE OR REPLACE TABLE Post_hasTag_Tag           AS FROM read_csv('PATHVAR/Post_hasTag_Tag.csv');
+CREATE OR REPLACE TABLE Forum_hasMember_Person    AS FROM read_csv('PATHVAR/Forum_hasMember_Person.csv');
+CREATE OR REPLACE TABLE Forum_hasTag_Tag          AS FROM read_csv('PATHVAR/Forum_hasTag_Tag.csv');
+CREATE OR REPLACE TABLE Message_hasCreator_Person        AS FROM read_csv('PATHVAR/Message_hasCreator_Person.csv');
+CREATE OR REPLACE TABLE Message_hasTag_Tag        AS FROM read_csv('PATHVAR/Message_hasTag_Tag.csv');
+CREATE OR REPLACE TABLE Message_isLocatedIn_Country        AS FROM read_csv('PATHVAR/Message_isLocatedIn_Country.csv');
+CREATE OR REPLACE TABLE Message_replyOf_Message        AS FROM read_csv('PATHVAR/Message_replyOf_Message.csv');
+CREATE OR REPLACE TABLE Person_isLocatedIn_City    AS FROM read_csv('PATHVAR/Person_isLocatedIn_City.csv');
+CREATE OR REPLACE TABLE Person_hasInterest_Tag    AS FROM read_csv('PATHVAR/Person_hasInterest_Tag.csv');
+CREATE OR REPLACE TABLE Person_likes_Comment      AS FROM read_csv('PATHVAR/Person_likes_Comment.csv');
+CREATE OR REPLACE TABLE Person_likes_Message     AS FROM read_csv('PATHVAR/Person_likes_Message.csv');
+CREATE OR REPLACE TABLE Person_likes_Post         AS FROM read_csv('PATHVAR/Person_likes_Post.csv');
+CREATE OR REPLACE TABLE Person_studyAt_University AS FROM read_csv('PATHVAR/Person_studyAt_University.csv');
+CREATE OR REPLACE TABLE Person_workAt_Company     AS FROM read_csv('PATHVAR/Person_workAt_Company.csv');
+CREATE OR REPLACE TABLE Person_knows_Person AS FROM read_csv('PATHVAR/Person_knows_Person.csv');
+CREATE OR REPLACE TABLE Post_hasCreator_Person AS FROM read_csv('PATHVAR/Post_hasCreator_Person.csv');
+CREATE OR REPLACE TABLE Post_isLocatedIn_Country AS FROM read_csv('PATHVAR/Post_isLocatedIn_Country.csv');
+CREATE OR REPLACE TABLE Tag_hasType_TagClass AS FROM read_csv('PATHVAR/Tag_hasType_TagClass.csv');
+CREATE OR REPLACE TABLE TagClass_isSubClassOf_TagClass AS FROM read_csv('PATHVAR/TagClass_isSubClassOf_TagClass.csv');
```

Comment:
The filename is:

```
TagClass_isSubclassOf_TagClass.csv
```

with a lower case `c` in `Subclass`.

Reply:
Updated this